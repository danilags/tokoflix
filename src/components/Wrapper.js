import React from 'react';
import {
  Container,
} from 'reactstrap';

const Wrapper = (props) => (
  <Container style={styles.container}>
    {props.children}
  </Container>
)

const styles = {
  container: {
    flex: 1,
    minHeight: '500px'
  }
}

export default Wrapper;
