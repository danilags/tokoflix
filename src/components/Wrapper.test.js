/**
 * Testing Wrapper component
 */

import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

import Wrapper from './Wrapper';

describe('<Wrapper />', () => {
  it('Component renders and matches snapShot', () => {
    let child = [
      <h3>Child 1</h3>,
      <p>Child 2</p>
    ];
    const wrapper = shallow(<Wrapper {...child} />)
    expect(wrapper).toMatchSnapshot();
  });

  it('test Wrapper matches MOUNT', () => {
    let inst = renderer.create(React.createElement('div', <Wrapper />));
    expect(inst.toJSON()).toMatchSnapshot();
  });
})
