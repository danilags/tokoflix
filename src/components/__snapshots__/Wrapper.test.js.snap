// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> Component renders and matches snapShot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Wrapper
    0={
      <h3>
        Child 1
      </h3>
    }
    1={
      <p>
        Child 2
      </p>
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": undefined,
      "style": Object {
        "flex": 1,
        "marginBottom": "3rem",
        "marginTop": "3rem",
        "minHeight": "500px",
        "padding": "3rem 0rem",
      },
      "tag": "div",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": undefined,
        "style": Object {
          "flex": 1,
          "marginBottom": "3rem",
          "marginTop": "3rem",
          "minHeight": "500px",
          "padding": "3rem 0rem",
        },
        "tag": "div",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`<Wrapper /> test Wrapper matches MOUNT 1`] = `
<div
  $$typeof={Symbol(react.element)}
  _owner={null}
  _store={Object {}}
  props={Object {}}
  type={[Function]}
/>
`;
